### DISCUSSION TITLE
Undefined reference to `vPortEvaluateYieldFromISR' when building custom FreeRTOS

### DISCUSSION THREAD

**Discussion by jahoyosr** [espressif/arduino-esp32#11098] [Q&A]
### Board

ESP32S3

### Device Description

I have a custom hardware that use the ESP32S3 module.

### Hardware Configuration

-

### Version

latest master (checkout manually)

### IDE Name

VSCode

### Operating System

Ubuntu 22.04

### Flash frequency

240MHz

### PSRAM enabled

yes

### Upload speed

115200

### Description

Hello,

I am trying to use the following FreeRTOS functions in my project:

vTaskList
vTaskGetRunTimeStats

Following the guidance in issue (https://github.com/espressif/arduino-esp32/issues/1350), I successfully created a custom FreeRTOS build with these functions enabled. However, I now encounter an undefined reference error for the function vPortEvaluateYieldFromISR.

Is there a specific configuration option that needs to be set in the sdkconfig file to include this function in the build?

Thank you for your assistance!

### Sketch

```cpp
-
```

### Debug Message

```plain
/xtensa-esp32s3-elf/bin/ld: .pio/build/esp32dev/libFrameworkArduino.a(esp32-hal-tinyusb.c.o):(.literal.hw_cdc_reset_handler+0x4): undefined reference to `vPortEvaluateYieldFromISR'
```

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by Jason2866** [comment]
This is Arduino github issues. Not an forum to ask questions.
